torch/package/package_exporter.py

```diff
@@ -1032,6 +1032,15 @@ def get_rdeps(self, module_name: str) -> List[str]:
         else:
             return []
 
+    def all_paths(self, src: str, dst: str) -> DiGraph:
+        """Return a subgraph that has all paths from src to dst.
+
+        Returns:
+            A DiGraph containing all paths from src to dst.
```

Comment:
I changed my mind about this return value. One of the issues is that `DiGraph` is not a public API and is typically in a format that doesn't make sense to external users. Can we use the same functionality that is in `dependency_graph_string` to return a graphviz string instead of a raw `DiGraph`?

Reply:
done